consensus/polybft/contractsapi/contractsapi_test.go

```diff
@@ -61,47 +50,44 @@ func TestEncoding_Method(t *testing.T) {
 func TestEncoding_Struct(t *testing.T) {
 	t.Parallel()
 
-	commitment := &StateSyncCommitment{
-		StartID: big.NewInt(1),
-		EndID:   big.NewInt(10),
-		Root:    types.StringToHash("hash"),
+	bridgeBatch := BridgeMessageBatch{
+		Messages:           []*BridgeMessage{},
+		SourceChainID:      big.NewInt(1),
+		DestinationChainID: big.NewInt(0),
 	}
 
-	encoding, err := commitment.EncodeAbi()
+	encoding, err := bridgeBatch.EncodeAbi()
 	require.NoError(t, err)
 
-	var commitmentDecoded StateSyncCommitment
+	var bridgeBatchDecoded BridgeMessageBatch
 
-	require.NoError(t, commitmentDecoded.DecodeAbi(encoding))
-	require.Equal(t, commitment.StartID, commitmentDecoded.StartID)
-	require.Equal(t, commitment.EndID, commitmentDecoded.EndID)
-	require.Equal(t, commitment.Root, commitmentDecoded.Root)
+	require.NoError(t, bridgeBatchDecoded.DecodeAbi(encoding))
+	require.Equal(t, bridgeBatch.SourceChainID, bridgeBatchDecoded.SourceChainID)
+	require.Equal(t, bridgeBatch.DestinationChainID.Uint64(), bridgeBatchDecoded.DestinationChainID.Uint64())
 }
 
 func TestEncodingAndParsingEvent(t *testing.T) {
 	t.Parallel()
 
 	var (
-		exitEventAPI      L2StateSyncedEvent
-		stateSyncEventAPI StateSyncedEvent
+		bridgeMsgEventAPI BridgeMsgEvent
 	)
 
 	topics := make([]ethgo.Hash, 4)
-	topics[0] = exitEventAPI.Sig()
+	topics[0] = bridgeMsgEventAPI.Sig()
 	topics[1] = ethgo.BytesToHash(common.EncodeUint64ToBytes(11))
 	topics[2] = ethgo.BytesToHash(types.StringToAddress("0x1111").Bytes())
 	topics[3] = ethgo.BytesToHash(types.StringToAddress("0x2222").Bytes())
-	someType := abi.MustNewType("tuple(string firstName, string lastName)")
-	encodedData, err := someType.Encode(map[string]string{"firstName": "John", "lastName": "Doe"})
+	someType := abi.MustNewType("tuple(string firstName,string secondName ,string lastName)")
+	encodedData, err := someType.Encode(map[string]string{"firstName": "John", "secondName": "data", "lastName": "Doe"})
 	require.NoError(t, err)
 
 	log := &ethgo.Log{
-		Address: ethgo.Address(contracts.L2StateSenderContract),
-		Topics:  topics,
-		Data:    encodedData,
+		Topics: topics,
+		Data:   encodedData,
 	}
 
-	var exitEvent L2StateSyncedEvent
+	var exitEvent BridgeMsgEvent
 
 	// log matches event
 	doesMatch, err := exitEvent.ParseLog(log)
```

Comment:
(Rename other occurences of `exitEvent` as well)
```suggestion
	doesMatch, err := bridgeMsgEvent.ParseLog(log)
```